src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -19,8 +20,8 @@
 public class SampleDataUtil {
     public static Budget[] getSampleBudgets() {
         return new Budget[] {
-            new Budget("Test Title 1", Arrays.asList(getSampleExpenditures())),
```

Comment:
I think this portion shouldn't be changed!

Reply:
Alright will update